src/parser.rs

```diff
@@ -80,23 +81,22 @@ named!(
 );
 
 named!(
-    num_range<Vec<i32>>,
+    num_range<Range<i32>>,
     do_parse!(
         from_: integer >> many1!(br) >> tag!("to") >> many1!(br) >> to_: integer
-            >> ((from_..(to_ + 1)).collect())
+            >> (from_..(to_ + 1))
```

Comment:
Probably `RangeInclusive` to avoid integer overflow.

Reply:
`RangeInclusive` is not stable. I'll add a todo.
In the meantime i'll put a `saturating_add`